aldryn_newsblog/models.py

```diff
@@ -255,7 +255,7 @@ class NewsBlogCMSPlugin(CMSPlugin):
     cmsplugin_ptr = models.OneToOneField(
         CMSPlugin, related_name='+', parent_link=True)
 
-    app_config = models.ForeignKey(NewsBlogConfig)
+    app_config = models.ForeignKey(NewsBlogConfig, verbose_name=_('Apphook configuration'))
```

Comment:
@czpython I wondered about that. This migration didn't seem to be created automatically.